projects/observability/src/shared/graphql/model/metadata/attribute-metadata.ts

```diff
@@ -7,6 +7,7 @@ export interface AttributeMetadata {
   units: string;
   type: AttributeMetadataType;
   scope: string;
+  category?: string;
```

Comment:
not generally applicable nor part of GQL model, please remove

Reply:
The category would be added to attributes eventually, right?

Reply:
I don't believe so - I thought the conclusion there was that concept was unique to a particular UI page and could be maintained there. It's not a universal concept across all attributes, nor is it necessarily universal that the same attribute belongs to the same category in all contexts - its use case specific.